WordPress/Classes/ViewRelated/NUX/LoginSegueHandler.swift

```diff
@@ -0,0 +1,9 @@
+protocol LoginSegueHandler {
+    associatedtype SegueIdentifier: RawRepresentable
+}
+
+extension LoginSegueHandler where Self: UIViewController, SegueIdentifier.RawValue == String {
```

Comment:
As this is login-specific, how about we limit it to `Self: NUXAbstractViewController`?